### ISSUE TITLE
[Spellstacking] Gbom/Battle Shout

### ISSUE THREAD

**Issue by HeadOwns** [dalaranwow/dalaran-wow#1193]
Hey there, I'll be direct, Greater Blessing of Might and Battle Shout, how it should work?

If u have BS and a pala use Gbom on u, it will analyse which one has more AP and it will take the one with more AP but both of them stay, lets say I have Gbom and a warrior used BS, both of the effects stay on me but it wont take both AP, only the one with the more AP depending on the talent of improvements.

How it works here?
If I have Gbom and a warrior uses BS, the Gbom effect is replaced by the BS effect regardless of the AP given from the Gbom (even if Gbom has more AP), this shouldn't work like this, both of them should stay but only take AP from the greatest one.

**Comment by robinsch** [comment]
This should count for all buffs - if proof is found.

**Comment by Rushor** [comment]
this was fixed with:

DB/Spell: Implement spell_group from Sunwell.pl

Issue https://github.com/dalaranwow/dalaran-wow/issues/1510
* commit: 424ed6af17475e0984dcda50ff1598607ef21796

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
I don't think this is on ptr, BoM/GboM and BS resplace each other which is not the intended behaviour.

**Comment by Aridam** [comment]
This is fixed but currently there is a different issue: same-Ability Buffs replace each other -- weaker Battle Shout replaces stronger Battle Shout, and so on; basically affects all cases in which same-Ability Buffs are used in this manner.
